src/main/java/com/photolocos/enterprise/dao/LocationDAOStub.java

```diff
@@ -14,21 +14,21 @@
  * LocationDAO class to handle Location related operations
  */
 @Component
-public class LocationDAO implements ILocationDAO {
+public class LocationDAOStub implements ILocationDAO {
```

Comment:
Added the Stub suffix to your stubs to differentiate between DAO Stubs and the final DAO implementation